src/main/scala/strawman/collection/Seq.scala

```diff
@@ -99,6 +99,32 @@ trait SeqOps[+A, +CC[X], +C] extends Any
    *          that the resulting collection has a length of at least `len`.
    */
   def padTo[B >: A](len: Int, elem: B): CC[B] = fromIterable(View.PadTo(toIterable, len, elem))
+  
+  /** Adds the element `sep` between each element of the sequence.
+    * e.g. `List(1, 2, 3, 4).intersperse(0) = List(1, 0, 2, 0, 3, 0, 4)`
```

Comment:
Could you use triple braces?

~~~
{{{
  List(1, 2, 3, 4).intersperse(0) == List(1, 0, 2, 0, 3, 0, 4)
}}}
~~~